pkg/init-lcow/Dockerfile

```diff
@@ -38,4 +38,5 @@ WORKDIR /
 COPY --from=mirror /out/ /
 COPY --from=build /out/ /
 COPY --from=runc /usr/bin/runc /sbin/runc
+RUN ln -s /run /var/
```

Comment:
please don't change `init-lcow`. It does things completely differently (completely different init)